paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheetViewModel.kt

```diff
@@ -238,9 +234,9 @@ internal class PaymentSheetViewModel @Inject internal constructor(
         )
     }
 
-    override val walletsProcessingState: StateFlow<WalletsProcessingState?> = viewState.map { viewState ->
+    override val walletsProcessingState: StateFlow<WalletsProcessingState?> = viewState.mapAsStateFlow { viewState ->
         mapViewStateToCheckoutIdentifier(viewState, CheckoutIdentifier.SheetTopWallet)
-    }.map { viewState ->
+    }.mapAsStateFlow { viewState ->
```

Comment:
Nit: Maybe a single map?

Reply:
Sure, I'll follow up with it!

Reply:
https://github.com/stripe/stripe-android/pull/8403